zephyr/modules/owntech_data_api/zephyr/public_api/DataAPI.cpp

```diff
@@ -110,11 +110,38 @@ void DataAPI::setParameters(channel_t channel, float32_t gain, float32_t offset)
 	data_conversion_set_conversion_parameters_linear(channel_info.adc_num, channel_info.channel_num, gain, offset);
 }
 
+float32_t DataAPI::getParameter(channel_t channel, parameter_t parameter_name)
+{
+	channel_info_t channel_info = shield_channels_get_enabled_channel_info(channel);
+	return data_conversion_get_parameter(channel_info.adc_num,channel_info.channel_num, parameter_name);
+
+}
+
+conversion_type_t DataAPI::getConversionType(channel_t channel)
+{
+	channel_info_t channel_info = shield_channels_get_enabled_channel_info(channel);
+	return data_conversion_get_conversion_type(channel_info.adc_num,channel_info.channel_num);
+}
+
+
+
+int8_t DataAPI::saveParametersInNVS(channel_t channel)
```

Comment:
NVS is a notion very specific to Zephyr. The name of this function should not include it, e.g. `storeParameters`.